.ci/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+test_repo="github.com/clearcontainers/tests"
```

Comment:
I assume this is temporary and we'll pull this into the `kata-containers/tests` correct? If so please add a `FIXME` to `TODO` in case we forget.